Services/Utilities/classes/class.ilUtil.php

```diff
@@ -1817,7 +1817,16 @@ public static function unzip($a_file, $overwrite = false, $a_flat = false)
 		ilUtil::execQuoted($unzip, $unzipcmd);
 
 		chdir($cdir);
-		
+
+		// remove all sym links
+		foreach (new RecursiveIteratorIterator(new RecursiveDirectoryIterator($dir)) as $name => $f)
+		{
+			if (is_link($name))
+			{
+				unlink($name);
```

Comment:
Maybe log the fact that a link was removed? I just thought that this might break stuff that uses links to other files in the same zip, so it would at least be nice to leave a note somewhere.